ncm-fstab/src/main/perl/fstab.pm

```diff
@@ -46,17 +46,42 @@ sub update_entries
     return %mounts;
 }
 
+# build the protected hash from the template
+sub protected_hash 
+{
+    my ($self, $config) = @_;
+
+    my $mounts_depr = $config->getElement("/software/components/fstab/protected_mounts")->getTree();
```

Comment:
This element in the tree is optional - what happens if it is not specified? Do you get a fatal caused by undef->getTree()?